composer.json

```diff
@@ -11,6 +11,7 @@
         "symfony/console": "4.3.*",
         "symfony/dotenv": "4.3.*",
         "symfony/expression-language": "4.3.*",
+        "symfony/filesystem": "4.4.*",
```

Comment:
Is this really usefull when you have the PHP filesystem functions ? https://www.php.net/manual/en/ref.filesystem.php

Reply:
yes very usefull